src/pkg/selectors/testdata/sharepoint.go

```diff
@@ -0,0 +1,9 @@
+package testdata
+
+import "github.com/alcionai/corso/src/pkg/selectors"
+
+// BackupFolderScope is the standard folder scope that should be used
+// in integration backups with sharepoint.
+func BackupFolderScope(sel *selectors.SharePointBackup) []selectors.SharePointScope {
```

Comment:
nit: should prefix this as `SharePointBackupFolderScope` as other services may need the same sort of function in the future